src/rendering/RenderUtil.cc

```diff
@@ -358,7 +358,9 @@ class ignition::gazebo::RenderUtilPrivate
   /// data.
   /// \sa actorManualSkeletonUpdate
   public: void UpdateAnimation(const std::unordered_map<Entity,
-              AnimationUpdateData> &_actorAnimationData);
+              AnimationUpdateData> &_actorAnimationData,
+              const std::unordered_map<Entity, math::Pose3d> &_entityPoses,
+              const std::unordered_map<Entity, math::Pose3d> &_trajectoryPoses);
```

Comment:
Can these new parameters be added to the method documentation as well?

Reply:
[Done](https://github.com/ignitionrobotics/ign-gazebo/pull/958/commits/0dd0970925853701625586b6b18073992248108b), I hope I did it right (position and content) since I'm not too familiar with this part of the code.

Reply:
Looks great, thanks!